### ISSUE TITLE
Navigation - Mobile Menu

### ISSUE THREAD

**Issue by D3Portillo** [Arena-Mayhem/frontend#8]
## TL;DR

Adapt Desktop Navigation for mobile devices

## Component Story

- As a user I want to be able to navigate in the `"Challenges"`, `"Character"`, `"Assets"` and `"Marketplace"` pages
- As a user I expect to see an "active state" when looking an specific page (turn "Challenges" text into yellow when navigating to root `"/"` page)
- As a user I want to connect/disconnect my Web3 Wallet
- As a user I want to be able to close the menu and open it back in mobile views
- As a user I expect to see the Desktop menu when in Desktop devices and hide the Mobile version

### Desktop

![Image](https://github.com/user-attachments/assets/65258a88-9776-4a04-8339-206dfd0a986c)

### Mobile

![Image](https://github.com/user-attachments/assets/3c6fcb49-9896-4f5d-a813-d64cd7f1659a)

## Resources

- Figma View: https://www.figma.com/design/csb1dd9yjNAg4sWTmVEZww/Arena-Mayhem?node-id=794-21673&t=UHndkZh79dxgcmJm-0

---

> [!TIP]
> You can use shadcn's UI to create this view. An example using Sheet Component: https://ui.shadcn.com/docs/components/sheet#examples

**Comment by cyanidev** [comment]
![image](https://github.com/user-attachments/assets/34c478ff-057c-4330-bf42-e30b864a3ef6)
![image](https://github.com/user-attachments/assets/78c5a22a-3672-4eac-9588-b233d4e25109)

**Comment by cyanidev** [comment]
before:

![image](https://github.com/user-attachments/assets/6af8df1c-52d6-4094-b393-906377badc80)

**Comment by D3Portillo** [comment]
closed by #11
